src/dlgNotepad.h

```diff
@@ -43,8 +43,14 @@ class dlgNotepad : public QMainWindow, public Ui::notes_editor
     void save();
     void restore();
 
+private slots:
+    void slot_text_written();
+
 private:
     QPointer<Host> mpHost;
+    bool mNeedToSave{};
+
+    void timerEvent(QTimerEvent *event);
```

Comment:
You are overriding a `virtual` method - do you need the `override` keyword at the end of this declaration?

Reply:
https://stackoverflow.com/a/39932616/4805858

Reply:
Yess fixed